samwell/overlap_detector.py

```diff
@@ -161,6 +161,20 @@ def get_overlaps(self, interval: Interval) -> List[Interval]:
             intervals = [i.data for i in tree.overlap(interval.start, interval.end)]
             return sorted(intervals, key=lambda intv: (intv.start, intv.end))
 
+    def get_enclosers(self, interval: Interval) -> List[Interval]:
```

Comment:
I am not loving this name, but I can't come up with a better one... Is there an App for this?